Dynatrace/dynatrace-operator
# Fix pod name prefix and shard id environment variable values.

## Description

The values of the environment variables `POD_NAME_PREFIX` and `SHARD_ID` for the OTel collector were set wrong.
 
## How can this be tested?

1. Deploy operator and dynakube with enabled extensions
2. Check env vars of pod extension-collector-x 
** POD_NAME_PREFIX = [dynakube-name]-extensions-collector
** SHARD_ID = (v1:metadata.labels['apps.kubernetes.io/pod-index'])
